### ISSUE TITLE
Can the data be provided?

### ISSUE THREAD

**Issue by abhikhanna30** [facebookresearch/VideoPose3D#10]
Hi,

Is it possible for you upload the data file dataset_path = 'data/data_3d_' + args.dataset + '.npz' for convenience?

Thanks

**Comment by dariopavllo** [comment]
Unfortunately we can't redistribute the dataset for license reasons.

You can set it up yourself using the provided scripts, assuming that you possess a valid copy of the dataset.

**Comment by Leopiglet** [comment]
Hello darlopavllo, there is a problem that though successfully getting "converted_15j" by Matlab, in next step using "data/prepare_data_humaneva.py" to get ".npz", the two produced files -- "data_2d_humaneva15_gt.npz" and "data_3d_humaneva15.npz" was so small (just 826B and 416B) that perhaps it caused the error when running "python run.py .... ".The detail is as follows:
Loading dataset...
Preparing data...
Loading 2D detections...
Selected actions: ['Walk', 'Jog', 'Box']
Traceback (most recent call last):
  File "run.py", line 169, in <module>
    cameras_valid, poses_valid, poses_valid_2d = fetch(subjects_test, action_filter)
  File "run.py", line 136, in fetch
    if parse_3d_poses and 'positions_3d' in dataset[subject][action]:
KeyError: 'Box 1 chunk0'

In short, I guess these two files,  "data_2d_humaneva15_gt.npz" and "data_3d_humaneva15.npz" , maybe have some unknown errors, please give me hand, thanks a lot!
